Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationOption.h

```diff
@@ -25,9 +25,12 @@ NS_ASSUME_NONNULL_BEGIN
 @interface BPKHorizontalNavigationOption : NSObject
 
 @property(nonatomic, readonly, strong, nullable) BPKIconName iconName;
+@property(nonatomic, readonly) NSInteger tag;
```

Comment:
What's the default value for `tag` if you don't use one of the constructors that sets it. Can we use `0` to align with `UIView.tag`?

Reply:
The default is `0` for a UIView that hasn't had tag set 👍

Reply:
But we set it unconditionally when creating the horizontal item. We never set the value of `tag` in `BPKHorizontalNavigationOption` when a constructor without tag is used

Reply:
Yeah but the tag value that is handed back comes from the UIView.

Reply:
And the default value for an NSInteger is `0` anyway

Reply:
> And the default value for an NSInteger is 0 anyway
This is not true, `NSIntger` in fact any primitive type is not guranteed to have a stable or deterministic value